### PR TITLE
Aligning lint rules with dart-pad

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: lib/src/compiler.dart [hunk 1]
```diff
@@ -95,7 +95,7 @@ class Compiler {
         return results;
       }
     } catch (e, st) {
-      _logger.warning("Compiler failed: $e /n $st");
```
Comment (domesticmouse) [inline lib/src/compiler.dart]: I doubt this was intended

File: lib/src/compiler.dart [hunk 2]
```diff
@@ -149,7 +149,7 @@ class Compiler {
...
```
Comment (domesticmouse) [inline lib/src/compiler.dart]: Nor this

File: tool/grind.dart [hunk 3]
```diff
@@ -94,19 +94,19 @@ void discovery() {
   File discoveryFile = File('doc/generated/dartservices.json');
   discoveryFile.parent.createSync();
   log('writing ${discoveryFile.path}');
-  discoveryFile.writeAsStringSync(result.stdout.trim() + '\n');
+  discoveryFile.writeAsStringSync('${result.stdout.trim()}\n');
 
   ProcessResult resultDb = Process.runSync(
       Platform.executable, ['bin/server_dev.dart', '--discovery', '--relay']);
 
-  if (result.exitCode != 0) {
```
Comment (domesticmouse) [inline tool/grind.dart]: This was constant false, due to prior throw. I'm guessing cut'n'paste fail.

**Comment by RedBrogdon** [comment]
This LGTM, but it might be a good one to hold off for a couple weeks until Devon is done with his changes, just to avoid slowing him down.

**Comment by devoncarew** [comment]
I can work around this, so no problems w/ landing it while I'm still working on my PR.

**Comment by RedBrogdon** [comment]
SGTM, then!

**Pull request merged**